app/views/users/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<h2 align="center">My Blocmarks</h2>
+  <div class="row">
+    <% @user_bookmarks_hash.each do |topic_id, bookmark_array| %>
+      <div class="col-xs-4">
+        <h3><%= Topic.find_by(id: topic_id).title %></h3>
```

Comment:
Its preferable to put database query code (e.g. `.where`, `.find_by` calls) in the controllers and models as it is simpler to write specs for them than it is to write specs for view templates.

Try to avoid database queries in view templates when possible.